bundles/edu.kit.kastel.sdq.eclipse.grading.api/src/edu/kit/kastel/sdq/eclipse/grading/api/controller/IAlertObserver.java

```diff
@@ -17,4 +17,6 @@
 	void info(String infoMsg);
 
 	void warn(String warningMsg);
+	
+	boolean confirm(String msg);

```

Comment:
I don't think that a "AlertObserver" should handle dialogs (if I look at its name)